### PR TITLE
Make sure values are non-empty before updating default user schema

### PR DESCRIPTION
**Pull request by geoffwhittington**

### CODE REVIEW

**Comment by JeffBain** [comment]
Mostly looks good here, just wasn't sure if we missed the odd logging statement or not. If those are intentional, +1.

File: sdetools/modules/sync_ldap/ldap_module.py [hunk 1]
```diff
@@ -897,9 +898,12 @@ def deactivate_stale_users(self, tracked_users, sd_users):
 
         for user in users_to_deactivate:
             try:
-                self.sde_plugin.update_user(**user)
+                self.sde_plugin.update_user(id=user['id'], is_active=False, groups=user['groups'])
                 deactivated_user_count += 1
 
+                error_msg = ("SD Elements user '%s' was not found in the "
+                             "directory and is now disabled")
+                logger.debug(error_msg, user["email"])
```
Comment (geoffwhittington) [inline sdetools/modules/sync_ldap/ldap_module.py]: self.emit() will trigger output in the celery logs. logger goes to stdout - so depending on how you run the command you get different output

Comment (geoffwhittington) [inline sdetools/modules/sync_ldap/ldap_module.py]: There is no "debug" statement for emit() so we want to be judicious on when we use emit

**Comment by geoffwhittington** [comment]
Great, thanks! You can merge and create a new release - I'm not sure how you guys are doing this anymore...

**Comment by geoffwhittington** [comment]
@JeffBain you didn't click squash and merge! I just assumed my noisy commits would go quietly into that good night... :)

**Pull request merged**